exercises/21_type_conversions/type_conversions_02.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * In this exercise, you will correct the issues in the given program.
+ * The program should convert all lowercase letters in a string to uppercase.
+ * Your task is to implement the correct logic in the `to_upper` function.
+ *
+ * Do not add any library.
+ */
+
+#include <assert.h>
+#include <stdio.h>
+#include <string.h>
+
+// ❌ I AM NOT DONE
+// hint: Take a look at ASCII table
+
+char to_upper(char c) {
+    if(c  'a' && c  'z') {  // <-- Fix here
```

Comment:
I think this will be really challenging for a beginner. I think we should give a little more information about ASCII tables and how conceptually you can derive the upper case letter from the lower case by using the number that it represents, and the pattern of 32 being the difference between them.

That being said it's a really cool example, I personally had not seen this before, so I think it'll be educational, I just don't want them to get stuck for too long.